src/ImageAndContent/Blocks/ImageAndContent.php

```diff
@@ -0,0 +1,263 @@
+     */
+    public $align_content = '';
+
+    /**
+     * The supported block features.
+     *
+     * @var array
+     */
+    public $supports = [
+        'align' => true,
+        'align_text' => false,
+        'align_content' => false,
+        'full_height' => false,
+        'anchor' => true,
+        'mode' => true,
+        'multiple' => true,
+        'jsx' => true,
+    ];
+
+    /**
+     * The block styles.
+     *
+     * @var array
+     */
+    public $styles = [];
+
+    /**
+     * The block preview example data.
+     *
+     * @var array
+     */
+    public $example = [];
+
+    /**
+     * The block view.
+     *
+     * @var array
+     */
+    public $view = 'Blocks.ImageAndContent.image-and-content';
+
+    /**
+     * Data to be passed to the block before rendering.
+     */
+    public function with(): array
+    {
+        $image = get_field('image');
+
+        return [
+            'canRenderBlock' => $this->canRenderBlock($image),
+            'blockClasses' => $this->getBlockClasses(),
+            'blockAnchor' => $this->getBlockAnchor(),
+            'wrapperSize' => get_field('wrapper_size'),
+            'verticalAlignment' => get_field('vertical_alignment'),
+            'horizontalAlignment' => get_field('horizontal_alignment'),
+            'image' => $image,
+            'alt' => get_field('alt'),
+            'imageDisplay' => get_field('image_display'),
+            'imagePlacement' => get_field('image_placement'),
+        ];
+    }
+
+    /**
+     * The block field group.
+     */
+    public function fields(): array
+    {
+        $imageAndContent = Builder::make('image_and_content');
+
+        $imageAndContent
+            ->addButtonGroup('wrapper_size', [
+                'instructions' => 'Set a different size for the content wrapper',
+                'wrapper' => [
+                    'width' => '33%',
+                ],
+            ])
+            ->addChoice('sm', 'Small')
+            ->addChoice('default', 'Default')
+            ->setDefaultValue('default');
+
+        $imageAndContent
+            ->addButtonGroup('vertical_alignment', [
+                'instructions' => 'Choose whether the content should align top, center or bottom',
+                'wrapper' => [
+                    'width' => '33%',
+                ],
+            ])
+            ->addChoice('top', 'Top')
+            ->addChoice('center', 'Center')
+            ->addChoice('bottom', 'Bottom')
+            ->setDefaultValue('center');
+
+        $imageAndContent
+            ->addButtonGroup('horizontal_alignment', [
+                'instructions' => 'Choose whether the content should align left, center or right',
+                'wrapper' => [
+                    'width' => '33%',
+                ],
+            ])
+            ->addChoice('left', 'Left')
+            ->addChoice('center', 'Center')
+            ->addChoice('right', 'Right')
+            ->setDefaultValue('center');
```

Comment:
These don't look great at 33% in the sidebar, could we change them to 100% or 50%?